Makefile

```diff
@@ -3,35 +3,47 @@
 # export CROSS_COMPILE=arm-none-linux-gnueabi-
 # make
 
+EXENAME = mcrcon
+PREFIX ?= /usr/local
+
+CFLAGS ?= -std=gnu99 -Wall -Wextra -Wpedantic -Os -s
+EXTRAFLAGS ?= -fstack-protector-strong
+
+CC ?= gcc
```

Comment:
It seems that on Windows my toolchain make (GNU Make 3.81) refuses to work if there is a question mark when defining CC variable.

I am getting following error: 
> process_begin: CreateProcess(NULL, cc -std=gnu99 -Wall -Wextra -Wpedantic -Os -s -fstack-protector-strong -o mcrcon.exe mcrcon.c -lws2_32, ...) failed.

Looks like the make uses inbuilt default value (which is 'CC') for CC so it fails to set correct value because of the `?=` operator (https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html).

I have little experience with makefiles so any idea would it be a good idea to just remove the question mark?
Is there any other workarounds?